src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Elasticsearch.Net" Version="5.6.0" />
-    <PackageReference Include="NEST" Version="5.6.0" />
+    <PackageReference Include="Elasticsearch.Net" Version="6.0.2" />
```

Comment:
Are NEST/Elasticsearch.Net 6.x backward-compatible with Elasticsearch 5 (server)?

If yes, please rev up the Outputs.Elasticsearch package version (`<VersionPrefix>` tag in the ElasticSearch.csproj file) to 2.4.0
If not, please rev up the package version to 3.0.0

Reply:
I did not want to push people to upgrade their elastic cluster. So I tested SendEventsAsync() data using Elasticsearch.NET 6.x client with a Elasticsearch cluster 5.6.9 in docker. Worked like a charm. =)

Updating to 2.4.0.